src/components/dropdown/interfaces.ts

```diff
@@ -170,7 +170,7 @@ export type DropdownProps<T> = {
   termSearchMessage?: ((term: string) => string) | string;
   /** Color variant of the dropdown */
   variant?: 'destructive';
-} & Omit<React.HTMLProps<HTMLInputElement>, 'onChange'> &
+} & Omit<React.HTMLProps<HTMLInputElement>, 'onChange' | 'value'> &
```

Comment:
Let's refactor this logic, in a `HTMLInputProps = Omit<React.HTMLProps<HTMLInputElement>, 'onChange' | 'value'>` for instance?